docs/source/howto/calculations/pw.rst

```diff
@@ -311,3 +311,38 @@ To retrieve additional output files, specify the list of files in the ``ADDITION
 
 .. |SSSP| replace:: Standard Solid-State Pseudopotentials (SSSP)
 .. _SSSP: https://www.materialscloud.org/discover/sssp/table/efficiency
+
+
+How to analyze the results
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When a ``PwCalculation`` is completed, there are quite a few possible analyses to perform.
+
+How to check the SCF accuracy during the self-consistent cycle
+..............................................................
+
+During the self-consistent field cycle, the difference in energy of the newly computed charge density and the starting one is computed and stored.
+It can easily be retrieved through the :meth:`~aiida_quantumespresso.tools.calculations.pw.PwCalculationTools.get_scf_accuracy` method.
+This method can be accessed directly through the node of a completed ``PwCalculation``:
```

Comment:
One final nitpick: I'd still mention the `tools` name space in this sentence, e.g.:

```suggestion
This method can be accessed directly through the ``tools`` of a completed ``PwCalculation`` node:
```